com.chartboost.mediation/Android~/chartboost-mediation-android-bridge/src/main/java/com/chartboost/mediation/unity/EventProcessor.kt

```diff
@@ -13,7 +15,35 @@ object EventProcessor {
         = eventConsumer.accept(placementName)
 
     @JvmStatic
-    fun serializeEventWithError(placementName: String, error: ChartboostMediationAdException?, eventConsumer: EventWithErrorConsumer<String, String>)
+    fun serializeLoadEvent(
+        placementName: String, loadId: String?, data: Map<String, String>?, error: ChartboostMediationError?,
+        loadConsumer: LoadEventConsumer<String, String, String, String, Double, String>) {
+        val errorMessage = error?.toString() ?: ""
+        var partnerId = ""
+        var auctionId = ""
+        var price =  0.0
+        val adLoadId = loadId ?: ""
+
+        data?.let { winningBidInfo ->
```

Comment:
Do we want to log in case `data` is null?

Reply:
Same comment as above, this is how is has been for a long time. I will take a look at it but in a separate PR.